pkg/controller/migration/namespace_migration.go

```diff
@@ -270,8 +270,10 @@ func (m *CoreNamespaceMigration) ensureTyphaRoom(ctx context.Context, log logr.L
 		// No Typha running - we're good to go.
```

Comment:
I think this check is bugged somehow - maybe it's not returning nil but instead just an empty struct?

Reply:
I would think we would hit this in more places than just here. Yeah it seems odd.